### ISSUE TITLE
Current Savestate Slot Hotkeys

### ISSUE THREAD

**Issue by trymeouteh** [project64/project64#1455]
It would be cool to be able to press a key to change the current savestate instead of pressing ~, 1, 2, 3, 4, 5, 6, 7, 8, 9, & 0 to change savestate slots. When you press this key it switches to the next savestate slot.

**Comment by dftf-stu** [comment]
Yes, this is a feature many emulators thesedays offer, but as of Dev-4.0.0.5871 is something I see _Project64_ still doesn't.

Under **Options** > **Configuration** > **Hotkeys** could you please consider adding "Previous save slot" and "Next save slot" into "Save slots" heading for both the "Game playing (windowed)" and "Game playing (full-screen)" CPU state categories?

In most of my current emulators, I have a setup like this:
- **F1:** save state
- **F2:** go to next save-state slot (e.g. from 1 to 2)
- **F3:** load state
- **F4:** go to previous save-state slot (e.g. from 2 to 1)

**Comment by project64** [comment]
the hot keys were original based off the sierra games, like kings quest (released 1980), I doubt many people are familiar with a 40 year old game. So I am not opposed to a better/more modern shortcuts

**Comment by dftf-stu** [comment]
@project64: yes, could be worth changing them, given this is _Project64_, not _[SCUMM-VM](https://www.scummvm.org/)_, where such defaults would make more-sense!  (Though it may upset some long-term users who are used to the current defaults.)

I'd always found it odd, personally, that _Project64_ defaulted to F5 to commit a save-state and F7 to re-load it, when virtually every-other emulator I've used defaults to F1 to save and F3 to re-load.  Good to know why!

**Comment by IronYoshi64** [comment]
I am so extremely use to Save State and Load State being on F5 and F7. My mind is so use to this. In other emulators I setup my hotkeys to this as well. But doesn't matter if its changed or not, I could just reset them back to what I like anyways. :)

**Comment by flagrama** [comment]
>I'd always found it odd, personally, that Project64 defaulted to F5 to commit a save-state and F7 to re-load it, when virtually every-other emulator I've used defaults to F1 to save and F3 to re-load. Good to know why!

TBH, it's annoying _they_ don't use the standard convention of "F1 = Help"

**Comment by dftf-stu** [comment]
@flagrama: you mean like how _Project64_ users F1 to launch the help?  Er, nope, seems to do a "Soft Reset" by-default, while a game is running.  (Or _nothing_ while a game isn't running!)

Not that there would be any help-files to launch anyway... the best you get are some tooltips in some areas of the app (e.g. in the Graphics Settings dialog) but not in others (e.g. literally _all_ of the options within the "Configuration" window or the "Edit Game Settings" dialog).
